programs/mango-v4/src/state/oracle.rs

```diff
@@ -12,9 +12,9 @@ use switchboard_v2::AggregatorAccountData;
 use crate::accounts_zerocopy::*;
 
 use crate::error::*;
-use crate::state::load_whirlpool_state;
+use crate::state::load_orca_pool_state;
 
-use super::orca_mainnet_whirlpool;
+use super::{load_raydium_pool_state, orca_mainnet_whirlpool, raydium_mainnet};
```

Comment:
importing the orca function and raydium function from different modules looks weird